MBBSEmu/HostProcess/ExportedModules/Majorbbs.cs

```diff
@@ -1351,6 +1360,31 @@ public ReadOnlySpan<byte> Invoke(ushort ordinal, bool offsetsOnly = false)
             return null;
         }
 
+        private void setcnf()
+        {
+            string optnam = GetParameterString(0, stripNull: true);
+            string optval = GetParameterString(2, stripNull: true);
+
+            // if updating a new value after an applyem, clear the dictionary and start anew
+            if (applyemCalled)
```

Comment:
Why wouldn't we do this in `applyem()`? I'm not seeing why we'd defer it to here.

Reply:
You can call applyem twice on different files, so if you group all your changes with setcnf you can shard, e.g.

```
setcnf(...);
setcnf(...);
applyem("bbsmajor.msg");
applyem("whatever.msg");

// resets now from scratch
setcnf(...);
```